onnxruntime/core/framework/orttraining_partial_executor.h

```diff
@@ -31,6 +33,7 @@ class PartialExecutor : public IExecutor {
  private:
   ORT_DISALLOW_COPY_ASSIGNMENT_AND_MOVE(PartialExecutor);
   PartialGraphExecutionState& state_;
+  OrtValueCache* cache_;
```

Comment:
>OrtValueCache* [](http://example.com/codeflow?start=2&length=14)

Please do not use raw pointers, instead use unique or shared ptrs to keep ownership/disposal in check.